client/src/helpers/APICalls/notification.ts

```diff
@@ -0,0 +1,47 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+import { Notification } from '../../interface/User';
+
+type existingNotification = Notification;
+
+export const getNotification = async (): Promise<Notification[]> => {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
+    credentials: 'include',
+  };
+  return await fetch(`/notification`, fetchOptions)
+    .then((res) => res.json())
+    .catch((err) => ({
+      error: { message: 'Unable to get notifications', err },
+    }));
+};
+
+export const updateNotification = async (eNotification: existingNotification): Promise<Notification[]> => {
+  eNotification.opened = true;
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
```

Comment:
The POST method is for resource creation, won't it be better using PATCH here?

Reply:
Did you mean PUT action ?

Reply:
in the recent commit, implemented using PATCH action